smarts/core/default_map_builder.py

```diff
@@ -19,13 +19,16 @@
 # THE SOFTWARE.
 
 import os
-from dataclasses import replace
-from pathlib import Path
+import sys
 from typing import NamedTuple, Optional, Tuple
 
+# pytype: disable=import-error
+
 from smarts.core.road_map import RoadMap
 from smarts.core.utils.file import file_md5_hash, path2hash
 
+# pytype: enable=import-error
```

Comment:
What was failing to import here?

Reply:
Its for an old import statement which i think is removed now. We can remove this now.